src/main/java/com/ibr/fedora/App.java

```diff
@@ -39,6 +46,37 @@ private App() { }
  * @param args
  */
 public static void main(final String[] args) {
+final Options options = new Options();
+
+final Option host = new Option("h", "host", true, "host address");
+host .setRequired(true);
+options.addOption(host);
+
+final Option user = new Option("u", "user", true, "user login");
+user.setRequired(false);
+options.addOption(user);
+
+final Option password = new Option("p", "password", true, "user's password");
+password.setRequired(false);
+options.addOption(password);
+
+final CommandLineParser parser = new BasicParser();
+final HelpFormatter formatter = new HelpFormatter();
+CommandLine cmd;
+
+try {
```

Comment:
The left-justification in the class is a little surprising.
Is there a reason for not using a more usual indentation scheme?

Note to self: The checkstyle should be configured to flag this.

Reply:
Currently, we have not added indented the lines because of the check style. If we indent the lines we get an error. The error is: Line contains a tab character, when we indent it. Should we indent the code and ignore the check style.

<error line="64" column="1" severity="error" message="Line contains a tab character." source="com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck"/>

Reply:
Agreed. Tabs are not allowed, but spaces are:
https://wiki.duraspace.org/display/FF/Code+Style+Guide

Reply:
I understand, no problem, we will make the indentation changes ASAP.